packages/owl-bot/test/handlers.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// import * as assert from 'assert';
+import * as assert from 'assert';
+import {describe, it, afterEach} from 'mocha';
+
+import {createOnePullRequestForUpdatingLock} from '../src/handlers';
+import {ConfigsStore} from '../src/database';
+import {dump} from 'js-yaml';
+import * as suggester from 'code-suggester';
+import {Octokit} from '@octokit/rest';
+
+import * as sinon from 'sinon';
+const sandbox = sinon.createSandbox();
+
+type Changes = Array<[string, {content: string; mode: string}]>;
+
+describe('handlers', () => {
+  afterEach(() => {
+    sandbox.restore();
+  });
+  describe('createOnePullRequestForUpdatingLock', () => {
+    it('updates .github/.OwlBot.lock.yaml if no pull request found', async () => {
+      const lock = {
+        docker: {
+          image: 'foo-image',
+          digest: 'sha256:abc123',
+        },
+      };
+      const expectedYaml = dump(lock);
+      let recordedURI = '';
+      // Mock the database helpers used to check for/update existing PRs:
+      const fakeConfigStore = ({
```

Comment:
Suppose later, another argument is added to the method `recordPullRequestForUpdatingLock()`.

These tests will continue to compile and pass.  We won't find the bug until runtime in production.  Compare that to creating a `class FakeConfigsStore implements ConfigsStore { }` and pressing `Ctrl+.` to let VSCode implement all the methods for you.